contracts/acl/ACL.sol

```diff
@@ -11,7 +11,7 @@ interface ACLOracle {
 
 
 contract ACL is IACL, AragonApp, ACLHelpers {
-    bytes32 constant public CREATE_PERMISSIONS_ROLE = bytes32(1);
+    bytes32 constant public CREATE_PERMISSIONS_ROLE = keccak256("CREATE_PERMISSIONS_ROLE");
```

Comment:
I guess this is more of a cosmetic change, since the `bytes32` value doesn't really matter (as long as it doesn't collide)?

Reply:
it was made to make artifact generation easier for contracts that follow this pattern